lib/sb3_schema.json

```diff
@@ -0,0 +1,363 @@
+{
+    "id": "https://scratch.mit.edu/sb3_schema.json",
+    "$schema": "http://json-schema.org/schema#",
+    "description": "Scratch 3.0 Project Schema",
+    "definitions": {
+        "optionalString": {
+            "oneOf": [
+                {"type": "string"},
+                {"type": "null"}
+            ]
+        },
+        "assetId": {
+            "type": "string",
+            "pattern": "^[a-f0-9]{32}$"
```

Comment:
Should we include A-F as well, just incase... Can't see how we'd end up with non-lower case hex values though...

Reply:
sounds like a good plan =]